Shell/Shell.cpp

```diff
@@ -337,6 +337,7 @@ int Shell::builtin_dirs(int argc, const char** argv)
 
 int Shell::builtin_exit(int, const char**)
 {
+    stop_all_jobs();
```

Comment:
Most shells I've used don't exit at first if you have active jobs. Instead they say something like "hey dude u have active jobs". To exit with active jobs, you have to type exit twice.

I think we should do something similar :)

Reply:
Sure, I'll have it say "hey dude u have %d active jobs" :^)